packages/publication_pdf.scss

```diff
@@ -0,0 +1,51 @@
+@import '../node_modules/nhsuk-frontend/packages/nhsuk';
+
+@page {
+    size: A4;
+    margin-top: 10mm;
+    margin-left: 10mm;
+    margin-right: 10mm;
+    margin-bottom: 40mm;
+
+    @bottom-right {
+        font-size: 0.8em;
+        content: "Page "counter(page) " of "counter(pages);
```

Comment:
I dislike the spacing here but that's a terrible reason to change it (and might be critical)

Reply:
Blame autoformatters - I also don't like it, but I don't care enough to try and figure out exactly which rule is changing it and create an override